XRTK-Core/Packages/com.xrtk.core/Interfaces/Providers/Controllers/IMixedRealityController.cs

```diff
@@ -37,6 +37,18 @@ public interface IMixedRealityController
         /// </summary>
         IMixedRealityControllerVisualizer Visualizer { get; }
 
+        /// <summary>
+        /// The controller's angular velocity, if any.
+        /// Angular velocity refers to how fast an object rotates or revolves relative
+        /// to another point.
+        /// </summary>
+        UnityEngine.Vector3 AngularVelocity { get; }
+
+        /// <summary>
+        /// The controller's velocity, if any.
+        /// </summary>
+        UnityEngine.Vector3 Velocity { get; }
```

Comment:
See comment for AngularVelocity property

Reply:
Add it